src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -663,6 +743,43 @@ private void ProcessReceivedBlob(byte[] buffer, int count, AsyncProtocolRequest
             StartSendBlob(buffer, count, asyncRequest);
         }
 
```

Comment:
I'm surprised more code hasn't changed / been deleted.  For example, the above function is still using AsyncProtocolRequest... I would expect that to go away entirely.

Reply:
more can go. and it will. I wanted some early feedback and check if I can pass all CI platforms.

Reply:
Great.

Reply:
I pruned ~ 700 lines for APM and dead renegotiation code.